bin/main.js

```diff
@@ -12,6 +12,7 @@ var AWS         = require('aws-sdk-promise');
 // These will exist in taskcluster-base, when we move to the next version.
 var config      = require('typed-env-config');
 var loader      = require('taskcluster-lib-loader');
+var tc_lib_app  = require('taskcluster-lib-app');
```

Comment:
Really, I'm not.. it's a lot easier to just include one bundle... Most changes aren't breaking anyways, so we just want to downstream components to pick them up slowly.

Actually, it doesn't really matter... Except we don't have good variable naming conventions when we include modules directly... At least we all know what `base` name is...